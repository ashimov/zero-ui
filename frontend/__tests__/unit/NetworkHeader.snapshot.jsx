import { render } from "@testing-library/react";
import NetworkHeader from "components/NetworkHeader";
import { testNetwork } from "./NetworkHeader.test";

it("renders HomeLoggedOut unchanged", () => {
  const { container } = render(<NetworkHeader network={testNetwork} />);
  expect(container).toMatchSnapshot();
});
