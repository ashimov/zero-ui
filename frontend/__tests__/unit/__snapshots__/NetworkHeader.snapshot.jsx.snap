// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders HomeLoggedOut unchanged 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-item"
  >
    <h5
      class="MuiTypography-root MuiTypography-h5"
    >
      <span>
        0d303702cd0f1fc6
      </span>
    </h5>
    <h6
      class="MuiTypography-root MuiTypography-h6"
      style="font-style: italic;"
    >
      <span>
        new-net-11166
      </span>
    </h6>
    <span>
      Test Network
    </span>
  </div>
</div>
`;
